test/unit-tests/MultiBridgeMessageSender.t.sol

```diff
@@ -632,4 +745,22 @@ contract MultiBridgeMessageSenderTest is Setup {
         vm.expectRevert(Error.SENDER_ADAPTER_NOT_EXIST.selector);
         sender.removeSenderAdapters(adapters);
     }
+
+    /// @dev if all message bridges fail (not pay enough fees)
+    function test_all_MULTI_MESSAGE_SEND_FAILED() public {
```

Comment:
nit: rename 
- given this is not just when all bridges fail
- naming style is inconsistent with other tests